docs/core/whats-new/dotnet-9.md

```diff
@@ -0,0 +1,84 @@
+---
+title: What's new in .NET 9
+description: Learn about the new .NET features introduced in .NET 9.
+titleSuffix: ""
+ms.date: 02/13/2024
+ms.topic: overview
+ms.author: gewarren
+author: gewarren
+---
+# What's new in .NET 9
+
+Learn about the new features in .NET 9 and find links to further documentation.
+
+.NET 9, the successor to [.NET 8](dotnet-8.md), has a special focus on cloud-native apps and performance. It will be [supported for 18 months](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) as a short-term support (STS) release. You can [download .NET 9 here](https://dotnet.microsoft.com/download/dotnet/9.0).
```

Comment:
```suggestion
.NET 9, the successor to [.NET 8](dotnet-8.md), has a special focus on cloud-native apps and performance. It will be [supported for 18 months](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) as a standard-term support (STS) release. You can [download .NET 9 here](https://dotnet.microsoft.com/download/dotnet/9.0).
```